covid_xprize/examples/predictors/lstm/xprize_predictor.py

```diff
@@ -256,7 +256,7 @@ def train(self):
         train_losses = []
         val_losses = []
         test_losses = []
-        for t in range(NUM_TRIALS):
+        for t in range(num_epochs):
```

Comment:
This should be `num_trials`.
`num_epochs` could be another parameter too, but it would be used a few lines below to replace `epochs=1000` by `epochs=num_epochs`. Default value could be 1000, but lower for the test to speed it up.

Reply:
Oh, oops! That's not right. Good catch, thank you very much.